include/bluetooth/audio/audio.h

```diff
@@ -1073,6 +1110,40 @@ struct bt_audio_unicast_server_cb {
 	 */
 	int (*publish_capability)(struct bt_conn *conn, uint8_t type,
 				  uint8_t index, struct bt_codec *const codec);
+
+	/** @brief Publish location callback
```

Comment:
I was wondering whether this should be be "published" rather than "publish"?
I think that would be more correct (these are callbacks to get the published parameters, not to publish them).
But at least it is consistent with the previous contents of the file.
(I also have a feeling that I have given a similar comment before ...)

Reply:
Ah, but all the characteristics in PACS are the "Published Capabilities".  And this is just a callback to read the value of one of them, right?  I think read_location would have been better.  But is OK as is.